src/main/scala/db/migration/V6__Populate_Subject_Id_column.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Part of NDLA article_api.
+ * Copyright (C) 2016 NDLA
+ *
+ * See LICENSE
+ */
+
+package db.migration
+
+import java.sql.Connection
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import scalikejdbc._
+import no.ndla.articleapi.ComponentRegistry.migrationApiClient
+
+class V6__Populate_Subject_Id_column extends JdbcMigration {
```

Comment:
Der og da syntes jeg det ble cleanere å spiltte det opp i to deler, men det er bedre å ha alt i samme migration script. Fikser det.

Jeg har ikke testet at det kjører i riktig rekkefølge, men tok det egentlig som en selvfølge :-) Navnet på filen inneholder jo tross alt versjonsnumre